src/Microsoft.Health.Dicom.Api/Controllers/QueryController.cs

```diff
@@ -43,6 +43,7 @@ public QueryController(IMediator mediator, ILogger<QueryController> logger)
 
         [HttpGet]
         [AcceptContentFilter(new[] { KnownContentTypes.ApplicationDicomJson }, allowSingle: true, allowMultiple: false)]
+        [ProducesAttribute(KnownContentTypes.ApplicationDicomJson)]
```

Comment:
@wsugarman I added these because in the generated swagger it was showing other accept types. Adding this fixes it but I want to make sure there arent any unintended consequences.

Reply:
`Produces` instead of `ProducesAttribute`?

Reply:
I moved adding these to a seperate pr